src/weapon/w_053.c

```diff
@@ -92,7 +92,7 @@ static void func_ptr_80170008(Entity* self) {
         self->palette = PAL_OVL(0x100);
         self->zPriority = PLAYER.zPriority;
         self->facingLeft = PLAYER.facingLeft;
-        self->animCurFrame = PLAYER.animCurFrame - ANIMSET_OVL_FLAG;
+        self->animCurFrame = PLAYER.animCurFrame - ANIM_FRAME_LOAD;
```

Comment:
Might be clearer as `+ ANIM_FRAME_LOAD` based on @hohle's findings. Otherwise looks good!